playwright-e2e/utils/test-utils.ts

```diff
@@ -16,7 +16,7 @@ export async function waitForNoSpinner(page: Page): Promise<void> {
 }
 
 export async function waitUntilRemoved(locator: Locator): Promise<void> {
-  expect(await locator.count()).toHaveLength(0);
+  await expect(locator).toHaveCount(0);
```

Comment:
See https://playwright.dev/docs/test-assertions#locator-assertions-to-have-count
`expect(locator)` waits until the expected condition is met or until time out is reached.